### PR TITLE
[3.2][ticket/8708] Add extra permission for posting global announcements

### PR DESCRIPTION
**Pull request by Elsensee**
Since this should go into 3.2 I decided to open a new pull request because the migration namespace and the merge target had to be changed.

Old PR: #2329 

https://tracker.phpbb.com/browse/PHPBB3-8708

### CODE REVIEW

**Comment by Nicofuma** [comment]
Can you rebase please?

**Comment by Elsensee** [comment]
Rebased. :)

**Comment by Nicofuma** [comment]
Being a forum based permission (and not a global one) looks a bit odd to me....
Are you sure it's better like that? (cc: @nickvergessen )

**Comment by nickvergessen** [comment]
Well global announcements are forum based, so they should be forum based permissions

File: phpBB/install/schemas/schema_data.sql [hunk 1]
```diff
@@ -309,6 +309,7 @@ INSERT INTO phpbb_config_text (config_name, config_value) VALUES ('contact_admin
 # -- Forum related auth options
 INSERT INTO phpbb_acl_options (auth_option, is_local) VALUES ('f_', 1);
 INSERT INTO phpbb_acl_options (auth_option, is_local) VALUES ('f_announce', 1);
+INSERT INTO phpbb_acl_options (auth_option, is_local) VALUES ('f_announce_global', 1);
```
Comment (nickvergessen) [inline phpBB/install/schemas/schema_data.sql]: I'd use f_global, so it's shorter and more in line with the others

Comment (Elsensee) [inline phpBB/install/schemas/schema_data.sql]: Do you want shorter code or more readable code?

Comment (Nicofuma) [inline phpBB/install/schemas/schema_data.sql]: bump

Comment (Nicofuma) [inline phpBB/install/schemas/schema_data.sql]: @nickvergessen shorter or longer? personally I don't care

**Comment by Nicofuma** [comment]
It needs to be rebased onto master

File: phpBB/install/schemas/schema_data.sql (hunk 1 above)
Comment (nickvergessen) [inline phpBB/install/schemas/schema_data.sql]: me neither

Comment (Nicofuma) [inline phpBB/install/schemas/schema_data.sql]: lets keep it like that in this case

**Comment by Elsensee** [comment]
This just looks horribly wrong here on GitHub but.. whatever.. rebased onto master. :D

**Comment by Nicofuma** [comment]
resend a new PR against master please :)

**Pull request closed without merging**